docroot/themes/humsci/su_humsci_admin/Gruntfile.js

```diff
@@ -44,7 +44,7 @@ module.exports = function(grunt) {
     uglify: {
       // This is for minifying all of our scripts.
       options: {
-        sourceMap: true,
+        sourceMap: false,
```

Comment:
@pookmish: Why no source map?

Reply:
it clogs up my dblogs with 404 for the .map files. does it not?

Reply:
Oh interesting. I have not seen that. @boznik have you seen this happening?

Reply:
<img width="971" alt="screen shot 2018-07-11 at 8 56 24 am" src="https://user-images.githubusercontent.com/7185045/42584590-623dba4e-84e8-11e8-84e1-518a200ba12c.png">